example/src/screens/Examples/ReanimatedChatFlatlist/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { FlatList, ListRenderItem, TextInput, View } from 'react-native';
+import { useReanimatedKeyboardAnimation } from 'react-native-keyboard-controller';
+import { useAnimatedStyle } from 'react-native-reanimated';
+
+import Message from '../../../components/Message';
+import { history } from '../../../components/Message/data';
+
+import type { StackScreenProps } from '@react-navigation/stack';
+import Animated from 'react-native-reanimated';
+import { MessageProps } from '../../../components/Message/types';
+import type { ExamplesStackParamList } from '../../../navigation/ExamplesStack';
+import styles from './styles';
+
+type Props = StackScreenProps<ExamplesStackParamList>;
+
+function ReanimatedChatFlatlist({ navigation }: Props) {
+  const { height } = useReanimatedKeyboardAnimation();
+
+  const fakeView = useAnimatedStyle(
+    () => ({
+      height: Math.abs(height.value),
+    }),
+    []
+  );
+
+  const renderItem: ListRenderItem<MessageProps> = ({ item, index }) => {
+    return <Message key={index} {...item} />;
+  };
```

Comment:
Can we make it a separate function outside of the render? Don't want to re-create this function on every re-render (thou I understand that in this example it'll not be recreated ever, because we are not using state or external props, but a lot of people may try to copy the code, so it would be better to create a proper design in advance 🙂).